.clang-tidy

```diff
@@ -21,10 +21,9 @@ Checks:
   modernize-use-bool-literals,
   modernize-use-default-member-init,
   modernize-use-nodiscard,
-  modernize-use-trailing-return-type,
   modernize-use-using'
 WarningsAsErrors: '*'
-HeaderFilterRegex: '".*/farm_ng/.*"'
+HeaderFilterRegex: cpp/farm_ng/.*
```

Comment:
This was wrong. It did not check any headers.

Now it should be setup in a way that it finds all headers within farm-ng-core, but not the autogenerated headers for the protos.